src/slack/slack_types.bal

```diff
@@ -0,0 +1,189 @@
+    string id;
+    string team_id;
+    string name;
+    boolean deleted;
+    string real_name;
+    string tz;
+    string tz_label;
+    Profile profile;
+    boolean is_admin;  
+    boolean is_owner;
+    boolean is_primary_owner;
+    boolean is_restricted;
+    boolean is_bot;
+    boolean is_app_user;
+    boolean has_2fa;
+};
+
+# Contains information about the user profile.
+# 
+# + title - Title added in the user profile
+# + phone - Phone number 
+# + skype - skype id
+# + real_name - The real name that the user specified in their workspace profile
+# + real_name_normalized - The `real_name` field, but with any non-Latin characters filtered out
+# + display_name - The display name that the user has chosen to identify themselves by in their workspace profile
+# + display_name_normalized - The `display_name` field, but with any non-Latin characters filtered out
+public type Profile record {
+    string title;
+    string phone;
+    string skype;
+    string real_name;
+    string real_name_normalized;
+    string display_name;
+    string display_name_normalized;
+};
+
+# Contains information about the file.
+# 
+# + id - File id
+# + created - Timestamp representing when the file was created
+# + name - File name
+# + title - File title
+# + user - The user id of the user who uploaded the file
+# + editable - `true` if the file is stored as editable
+# + size - File size in bytes
+# + mode - Contains one of hosted, external, snippet or post
+# + is_external - `true` if the master copy of the file is stored within the system, otherwise `false`
+# + external_type - Kind of external file eg: dropbox or gdoc
+# + is_public - `true` if the file is public
+# + public_url_shared - `true` if the file's public url has been shared
+# + url_private - Points to a URL to the file content
+# + url_private_download - Includes headers to force a browser download
+# + permalink - The URL pointing to a single page for the file containing details, comments and a download link
+# + permalink_public - Points to a public file itself
+# + lines - The total count of the lines shown in the snippet
+# + lines_more - The count of the lines not shown in the preview
+# + comments_count - The number of comments attached to the file
+public type FileInfo record {
+    string id;
+    float created;
+    string name;
+    string title;
+    string user;
+    boolean editable;
+    int size;
+    string mode;
+    boolean is_external;
+    string? external_type;
+    boolean is_public;
+    boolean public_url_shared;
+    string url_private;
+    string url_private_download;
+    string permalink;
+    string permalink_public;
+    int lines;
+    int lines_more;
+    int comments_count;
+};
+
+# Holds the parameters used to create a `Client`.
+#
+# + oAuth2Config - OAuth2 client configuration
+# + proxyConfig - Proxy configuration if connecting through a proxy
+public type Configuration record {
+   oauth2:DirectTokenConfig oAuth2Config;
```

Comment:
Shall we rename this as follows? If, so please update the markdown files as well.
```suggestion
   oauth2:DirectTokenConfig oauth2Config;
```